src/cephadm/cephadm

```diff
@@ -924,38 +924,47 @@ class CephExporter(object):
     port_map = {
         'ceph-exporter': 9926,
     }
+    DEFAULT_PORT = 9926
```

Comment:
sorry I didn't see this one in the first review. To avoid repeating the port you can change the order:

```
    DEFAULT_PORT = 9926
    port_map = {
        'ceph-exporter': DEFAULT_PORT,
    }

```

Reply:
Yes, thanks!